python/tank/platform/software_launcher.py

```diff
@@ -304,50 +305,53 @@ def _format(self, template, tokens):
         return template
 
     def _glob_and_match(self, match_template, template_key_expressions):
-        """
+        r"""
         This is a helper method that can be invoked in an implementation of :meth:`scan_software`.
 
         The ``match_template`` argument provides a template to use both for globbing files and then pattern
         matching them using regular expressions provided by the ``tokens_expressions`` dictionary.
 
-        The method will first substitute every token surrounded by `{}` from the template with a ``*``
+        The method will first substitute every token surrounded by ``{}`` from the template with a ``*``
         for globbing files. It will then replace the tokens in the template with the regular expressions
         that were provided.
 
         In the following example::
 
             self._glob_and_match(
-                "//network/softwares/Nuke{full_version}/Nuke{major_minor_version}.exe",
+                "\\\\network\\softwares\\Nuke{full_version}\\Nuke{major_minor_version}.exe",
```

Comment:
I suggest just changing this example (and subsequent references) to this:

```
r"C:\Program Files\Nuke{version}\Nuke{major_minor_version}.exe"
```

I think that would make it much clearer and also match the actual path used in the code. You also wouldn't need to worry about the docstring being a raw string.